zh-Hans.lproj/Localizable.strings

```diff
@@ -15,21 +15,21 @@
 "Latest" = "最新版";
 "Specific" = "特定版本";
 
-"Please confirm that you have adequate free storage before proceeding:" = "请确认您有足够的存储空间再进行操作:";
-"Choose the backup you'd like to restore from:" = "请选择您要恢复的备份:";
-"Are you sure that you want to restore from %@?" = "您确定要从以下备份进行恢复吗 %@?";
+"Please confirm that you have adequate free storage before proceeding:" = "请确认您有足够的存储空间再进行操作：";
+"Choose the backup you'd like to restore from:" = "请选择您要恢复的备份：";
+"Are you sure that you want to restore from %@?" = "您确定要从以下备份进行恢复吗 %@？";
 "Yes" = "是";
 "No" = "否";
-"Please note:" = "请注意:";
+"Please note:" = "请注意：";
 "You have chosen to restore from a developer backup. This backup includes bootstrap packages." = "您选择了从开发者备份进行恢复。该备份包括引导包。";
 "Please confirm that you understand this and still wish to proceed with the restore" = "请确认您已理解并仍然希望继续恢复";
 
 "Confirm" = "确认";
 "Cancel" = "取消";
 
-"Tweak backup completed successfully in %@ seconds!" = "插件备份已完成，用时 %@ 秒!";
+"Tweak backup completed successfully in %@ seconds!" = "插件备份已完成，用时%@秒!";
```

Comment:
The "%@" will be replaced by a number. Is this formatting still correct?

Reply:
Under the habit of using Chinese, I didn't add a space between Chinese and numbers, so I deleted the space.